common/flatpak-context.c

```diff
@@ -1023,6 +1023,67 @@ flatpak_context_take_filesystem (FlatpakContext        *context,
   g_hash_table_insert (context->filesystems, fs, GINT_TO_POINTER (mode));
 }
 
+static gboolean
+flatpak_context_add_conditional (FlatpakContext  *context,
+                                 const char      *string,
+                                 const char     **names,
+                                 GHashTable      *conditionals,
+                                 GError         **error)
+{
+  guint32 bitmask;
+  g_auto(GStrv) tokens = NULL;
+  g_autoptr(GPtrArray) conditions = NULL;
+  int i;
+
+  tokens = g_strsplit (string, ":", -1);
```

Comment:
I think this syntax on the commandline is unnecessarily complex.
 `--nodevice-if=all:has-input:has-usb` is already equivalent to `--nodevice-if=all:has-input --nodevice-if=all:has-usb`, and I'm not sure having the combined version is useful enough to need special handling.